pliers/tests/extractors/brain_decoding_text_encoding_interface.py

```diff
@@ -0,0 +1,277 @@
+import warnings
```

Comment:
Can you elaborate on the purpose of this file? Does it add testing functionality beyond what is covered in your other unit test file, `test_text_embeddings.py`?

Reply:
@qmac, sorry this file was a part of a project run by MIT+NIMH and it should not be here. I am making a note and will remove it. Thanks.